frame/staking/src/testing_utils.rs

```diff
@@ -34,6 +34,49 @@ use sp_std::prelude::*;
 
 const SEED: u32 = 0;
 
+// A helper enum to configure a payout destination without knowing the stash and controller
+// accounts beforehand.
+#[derive(PartialEq, Copy, Clone)]
+pub enum PayoutDestinationOpt<AccountId> {
+	Stake,
+	Controller,
+	Account(AccountId),
+	Split((Perbill, PayoutSplitOpt)),
+}
+
+// Options for splitting payouts. These are used to alias the stash and controller accounts, which
+// are not know at the time at usage.
+#[derive(PartialEq, Copy, Clone)]
+pub enum PayoutSplitOpt {
+	Stash,
+	Controller,
+}
+
+impl<AccountId> Default for PayoutDestinationOpt<AccountId> {
+	fn default() -> Self {
+		PayoutDestinationOpt::Stake
+	}
+}
+
+// A helper for converting a `PayoutDestinationOpt` value into a `PayoutDestination` value.
+pub fn opt_to_payout_destination<AccountId: Clone>(
```

Comment:
Same comment as above -- implementing `From<PayoutDestinationOpt>` for `PayoutDestination` should do the trick.

Reply:
I don't believe `From` is viable here because we still need to pass in the `stash` and `ctlr` variables. It is not a pure conversion.